pkg/talosclient/client.go

```diff
@@ -171,6 +171,7 @@ func NodeIPDiscovery(nodeIPs []string, ifaces []network.AddressStatusSpec) (publ
 		if iface.LinkName == constants.KubeSpanLinkName ||
 			iface.LinkName == constants.SideroLinkName ||
 			iface.LinkName == "lo" ||
+			iface.LinkName == "cilium_host" ||
```

Comment:
hmm I don't like having to be dependent on cilium, but i guess it is what it is :sweat_smile:

Reply:
Yeah, me neither )